Header header
float32 x
float32 y
float32 z
float32 dot_x
float32 dot_y
float32 dot_z
